.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,13 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\website",
+    "\\website\\common",
+    "\\website\\common\\locales",
+    "\\website\\common\\locales\\en",
+    "\\website\\common\\script",
+    "\\website\\common\\script\\content"
+  ],
+  "SelectedNode": "\\website\\common\\locales\\en\\limited.json",
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
@Inshabel: Also, files like this shouldn't really get added to the project, because they're configuration files for your editor and probably won't be very helpful for other people.